fibonacci.rb

```diff
@@ -0,0 +1,14 @@
+def fibonacci(length)
+  list_of_fibs = [1]
+  if length == 1
```

Comment:
Everything looks good to me. Since the block inside the if statement is only one line long, I'd shorten it to "return list_of_fibs if length == 1"